src/selection/style.js

```diff
@@ -16,7 +16,10 @@ d3_selectionPrototype.style = function(name, value, priority) {
     }
 
     // For style(string), return the computed style value for the first node.
-    if (n < 2) return d3_window.getComputedStyle(this.node(), null).getPropertyValue(name);
+    if (n < 2) {
+      var node = this.node();
+      return node.ownerDocument.defaultView.getComputedStyle(node, null).getPropertyValue(name);
```

Comment:
I appreciate this might be for performance, but should `d3_window(node)` be used here instead of `node.ownerDocument.defaultView`?